index.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
I think we can remove `index.php`. Tests use `tests/router.php` anyway.

Reply:
> I think we can remove `index.php`. Tests use `tests/router.php` anyway.

yes, it's not really needed, am removing it.



> WIth that said, let's please create `README.md` and have a quick usage example in there.

sure, I am working on it!